src/expr.rs

```diff
@@ -1860,7 +1860,10 @@ fn rewrite_let(
     // TODO(ytmimi) comments could appear between `let` and the `pat`
 
     // 4 = "let ".len()
-    let pat_shape = shape.offset_left(4)?;
+    let mut pat_shape = shape.offset_left(4)?;
+    if context.config.version() == Version::Two {
+        pat_shape = pat_shape.sub_width(2)?; // 2 to account for the length of " ="
```

Comment:
Nit: I'd prefer the comment to be above the line like we do in most other explanatory comments within the codebase.